src/amo/pages/UserProfileEdit/index.js

```diff
@@ -926,19 +951,19 @@ export function mapStateToProps(state: AppState, ownProps: Props) {
   const isReviewer = hasAnyReviewerRelatedPermission(state);
 
   return {
-    clientApp,
+    clientApp: clientApp || '',
```

Comment:
is this needed even with the changes from a previous PR?

Reply:
Nope. I've removed it.